src/tools/dartdoc.md

```diff
@@ -22,6 +22,7 @@ Documenting my_app...
 Success! Docs generated into /Users/me/projects/my_app/doc/api
 ```
 
+To work correctly, your package must [analyze][] without errors.
 By default, the documentation files are static HTML files,
```

Comment:
What do you think about using a note alert, as this seems to be pretty important for users to realize.

I'm not exactly sure the best location on the page for it, but it would look something like:
![image](https://user-images.githubusercontent.com/18372958/133136788-80f6b858-6435-4f29-a218-61dcd047ae0c.png)

Added like:

```suggestion
{{site.alert.note}}
  To work correctly, your package must [analyze][] without errors.
{{site.alert.end}}

By default, the documentation files are static HTML files,
```

Reply:
I don't object, will add if @mit-mit agrees

Reply:
I like it, but perhaps to make it even more concrete can we s/must [analyze][]/must pass [`dart analyze`][analyze]/ ?

Reply:
> I like it, but perhaps to make it even more concrete can we s/must [analyze][]/must pass [`dart analyze`][analyze]/ ?

Sure.  Also changed the wording as "to work correctly" is a bit vague looking when extracted into a note.